mrpython/configHandler.py

```diff
@@ -584,7 +584,7 @@ def GetCoreKeys(self, keySetName=None):
         keyBindings={
             '<<copy>>': ['<Control-c>', '<Control-C>'],
```

Comment:
I don't really understand the effect of these definitions... I commented out all the clipboard management stuff and it still works on linux... I'll push the change and we'll see if it works on the other OSes.